platform/platform-commands/src/main/java/org/codice/ddf/commands/platform/ExportCommand.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.platform;
+
+import org.apache.felix.gogo.commands.Command;
+import org.codice.ddf.configuration.store.ConfigurationMigrationService;
+
+/**
+ * Executes the export method in {@link ConfigurationMigrationService}.  Configurations
+ * are exported to the directory specified in the implementation of the service
+ * (see {@link ConfigurationFileDirectory} for an example).
+ */
+@Command(scope = PlatformCommands.NAMESPACE, name = "config-export", description = "Exports configurations")
```

Comment:
I think we can provide the option in the future, but it wasn't in the original design we came up with to allow specifying the output directory.  

I'm not sure we should add a console print, since it's possible that export can get called from something other than the CLI.  What do you think?